tests/test_styles.py

```diff
@@ -11,7 +11,7 @@
 
 """
 To test run:
-py.test --mpl
+b
```

Comment:
typo

Reply:
That's weird, most of my typos like this are `:w` :) Thanks!

Reply:
By the way, "pytest" is the recommended entry point for PyTest since version 3 (though py.test may remain around forever). The old `py.test` naming was from an era when they planned to have several things in a `py` namespace; `py.path` has been mostly replaced by pathlib and is no longer recommended standalone, and the focus is on pytest, so they dropped py.test from the docs and entry points by version 3.